doc/dev/tma/example1.svg

```diff
@@ -0,0 +1 @@
t/p27iC8yym5Bjk6x98vvLzuZh3yaT2Z3ZyVpJSUlJSUlJSTzWT24Hw2HvrI5/fxSs8ykDv7o1GD8Ep5DtM+2sw2w+1fkdB3ebGDGdLQheMO1UMGY2deilDLYxaDa/IZfinlsYNRj7D9SK9Dcwbiz2E6Qu83aIKwzFeoRQBU9GB8hrqFTy9hVXGcgxNHLc7/iDcm3jQtNoulDIOVxrveBPwnAHl5qF8wp9nDPPcLS8npoaH2+gjnP9bmmQ/8uMZ/N2YXuQxnmZrfe1HiyEUQuDptBcuMZ0CzYeJhXRxj3GoBlsjKCLIyg7Jq/tjBuDnN35C1GcB9hAYwg74dGc8HgPSZzp8hbFucMIYWDKe0oToVPYCG05PBoifZf48rPKh7d/YXSBEdLrE6jhjNSCKt8wvsAA6VXydrpNGCV233DJnMKlW3S1UTj5jE3JY4qWfgUdHr2wQG3/xFVzipVOFn32tiJ56OK6OUVK3yRvJ/sMYyCHywtqcdIrdBN+DmMINH76FCadBrtQJ5/x1RDpnzG9xziaBDOk18gGkX2BcRVGSH/A3B5IgyJggPQTzOwxjLFvLVw63a4Er/oqCpZuDzCth5/rR+cTXQU8cpVO1/KXuOnZToHSjzClhxu/QF6c9G2a5nRhjENR0m2yJ4/vKz6y9EWxI0NuMZtHAl/xkaSPcpDexlw+IdlEOAVIb9B4/JTIV3xk6Q2MZIRDaz+TNCdAkvTJJ4xkg1D5SXf+I0l39zCSBcL5+D2MSZGkszZG9LNDnXxchTUxsvSYO4nIrI8xgc8urCmQpWdT6hWqK+wS1lS0JOn9LA4gBScf6OlFkO/6eGUVITb0+C3x4ikhS3c7GNKFLVTxT2BNT5MmtO/80Hs4cImf9enBqIMNmqa8o9XZW/hRn4nW16gmn/uONAQuIPqK5hM2VZfGjS6PEXwl7eIpYdGdM9B02k4LWivLtkmQzzUYu069SIvH+owdwKoVRaMAG6efSXiYjzBqRnV2ynopO6n28Ts+blbJi0MLfjMmR2k2MtvCMhehUp6UM0wh0I9VQhOoCo/xNcvD731pK+Bxk/RNFR9i1IJzMhryVoAzTpZ8Cbk+u4U1K6qKyM55TpBeC2lQJqF8iQOlx7iXcctgttjUdARzlmwompE4k068KEPPahm7gzVj9pTezl7i5ATCqs9eNSzHkXDUbZtuN/Jtt2jxk7k5dhzuKlrvOP2ovfi0UM6fFqy54FwtH8v4TE4j5b81sbsjj9eTUFf3Vo/bEVzmBy5+J6sqTjCHUl+MT39l16P4emqpr8SkSnpPKb3wXR9tQ+SZoQNzvrTk9NrDvQpbEsUFIbtiZTiVjtplJsFZhxhXRsV9XuNcKNIlTuC9FB9TznFFZKsPFSIBlQ3xs6ZkR3DasLqq4B6wvojOUnj3dVP1pqq3j0LHmPYCS3wqZ/JtV+d5NKHIbacVSl367EcdGclHcPHaQDKkKWwA2QjmZRZplb6Kc1oqbZrNB4XFWZlyao5wjnU6X5huA+Od3R3zJ9I17Xu92rl/zja9CA3UtWI2KyuwGscn+/l185SUlJSUlPzHrK39AyMNoSCmBlQEAAAAAElFTkSuQmCC" preserveAspectRatio="none" id="img60"></image><image width="146" height="269" xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAJIAAAENCAMAAADT4syeAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAADPUExURQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAJljpgQAAABEdFJOUwAECAwQFBgcICQoLDA0ODxAREhMUFRYXGBkaGxwdHh8gIOHi4+Tl5ufo6Snq6+zt7u/w8fLz9PX297f4+fr7/P09/v+voEZfQAAAAlwSFlzAAAywAAAMsABKGRa2wAACi9JREFUeF7tXYdy2zgQleQSp/eeXHq79MSJc7Etn3X4/286kHiLBSSLBIgFwJnozdycACbiC8uWtwtossEGG2ywwQZ/JmZP3j7fxudxYOe3Ump+EaNR4IVmpNRHjEaBk5aSGtGtu24YqT2MRwBz35TawngE+GwY/cRwDDg2lF5gOALsGUbqIcYjwG1QuobxCPASlHYxHgG+GEZHGI4BPwylrxiOAUejo3TOMFJ/YzwC0Av37w1M1McTUFLz0fg4sgFKfcFMdbwFIY2xRCffwEepxVgoHYKQUocjiU62wOc//d9TzFXGDcOoxXwcd+456LR4i8m6+A42BtcxWxXwcMAPzNbEFFwItzBfERTlEkZgwa+Cyj/4/2IHB+rhHqhYT/cIB+rhGZhcbZSKBkczHKmGd2Cya73vXRypBpil+eSm+aDUBxypBmS6+5PJgfmk5lMcqoQd8NCX5iE+1k4xr4DG88nkAj6qNzhWCfdB44H+jHuojuveOZKWGndLL5+6bY5VwiewOKc/38Fn9d0cqwS8ZifN561TM1DqcnusEsDBxCT2ztUUvygOeN+O6PVTv9phHVwCB1yWXxiq82ZcAyQvPzFDev9oXAMUmtwzQ2JY8849BgWIJjOUCXSsYiYqgDKmSxi/wbhi9kRBEimn9s6dVHMqUE4VhpMp+bl6CR2M928MnTtXLRyAC2GnZmPLQ0yUBimnxng3mFk/V+mdoyTuGcYaqDq1QV0NXMPp72Os8QBTtawlUYLxbkD3slYIfhdn
```

Comment:
Could you add a white background to this picture? Just add a `style="background-color:white"` to `<svg>`. Also, could you add copyright info? See https://github.com/NVIDIA/Fuser/blob/4d45883df7cb6656f8b099c794aa740e527f1348/doc/dev/tma/arbitrary-view.svg#L1-L6